app/models/comment.rb

```diff
@@ -0,0 +1,11 @@
+class Comment < ApplicationRecord
+  validates :text, presence: true
+  belongs_to :post
+  belongs_to :user
+
+  private
+
+  def change_comment_counter
+    post.increment!(:comments_counter)
+  end
```

Comment:
**[OPTIONAL]**

It would be nice if you use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#change_comments_counter` method every time a new post has been saved to the database, then make the `update_post_counter` method private

```ruby
  after_save :change_comments_counte

  private

  def change_comment_counter
    post.increment!(:comments_counter)
  end
```